src/styles/blocks/card/card.css

```diff
@@ -0,0 +1,77 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  
+}
+
+.card__img {
+  display: block;
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__title-price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+
+}
+
+.card__number-price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__label {
+  background: #00acdc;
+  border-radius: 5px;
+  line-height: 40px;
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
+  display: flex;
+  justify-content: center;
+  border: 1px solid transparent;
+
+}
+
+.card__label:hover {
+  background: #fff;
+  font-weight: 700;
+  font-size: 14px;
```

Comment:
Do you need to set these properties here? Looks like they're not changed on hover state